testing/sdk_install.py

```diff
@@ -181,7 +181,7 @@ def get_package_options(additional_options={}):
     if os.environ.get('SECURITY', '') == 'strict':
         # strict mode requires correct principal and secret to perform install.
         # see also: tools/setup_permissions.sh and tools/create_service_account.sh
-        return _merge_dictionaries({
+        return merge_dictionaries({
```

Comment:
Perhaps better to move this method to a utility module? Otherwise seems like we'd just be going against convention of having a single underscore to indicate private methods?

Reply:
The prefix was rightly applied. This is a private method by all accounts. I recommend restoring it.

Reply:
#1683 also requires this change. I'll make the tweak there (as this is about to pass CI)

Reply:
(The tweak == move to utility function)